src/server/public/js/x5gon-log.js

```diff
@@ -50,13 +50,12 @@ var x5gonActivityTracker = function () {
   function x5gonGetReturnString(userId, providerToken) {
     var Dat = new Date();
     var Dt = Dat.getFullYear() + "-" + x5gonCheckTime(Dat.getMonth() + 1) + "-" +
-      x5gonCheckTime(Dat.getDate()) + " " + x5gonCheckTime(Dat.getHours()) + ":" +
-      x5gonCheckTime(Dat.getMinutes()) + ":" + x5gonCheckTime(Dat.getSeconds()) +
-      "." + Dat.getMilliseconds();
+      x5gonCheckTime(Dat.getDate()) + "T" + x5gonCheckTime(Dat.getHours()) + ":" +
+      x5gonCheckTime(Dat.getMinutes()) + ":" + x5gonCheckTime(Dat.getSeconds()) + "Z";
     var CURL = document.URL;
     var PURL = document.referrer;
 
-    var b = 'http://localhost:7110/api/v1/log?uid=';
+    var b = 'http://pankretas.ijs.si:7110/api/v1/log?uid=';
```

Comment:
2 things in this line:
1. in the long run I would suggest that we store URL in config file instead of having them hardcoded - this makes the code more portable (for example, if we will have to move the service to the Post Office servers, we would only have to change config file locally and everything would work fine, whereas having URLs hardcoded you would have to change every single one in code)

2. what is b? (I can see from the context, but still :) )

Reply:
This file will be included on the client websites meaning we cannot have the URL in a separate config file.